docs/installation/development.rst

```diff
@@ -104,6 +92,15 @@ Running tests
 See :doc:`../development/testing`
 
 
+Working on the theme
+--------------------
+
+The theme is derived from a separate package: ploneintranet.prototype.
+
+Documentation TODO.
+https://github.com/ploneintranet/ploneintranet.theme/blob/0ba4c21acd6da8e941fa94d0421491e59228009d/README.rst
+
+
 Building the Documentation
 --------------------------
 
```

Comment:
I think we should also document the steps for doing Javascript development.

Looking at the Makefile, those steps appear to be:
```bash
make jsdev # Checks out the prototype, makes a local javascript bundle, builds jekyll and copies HTML files to Diazo dir
# When ready to release JS changes:
# (perhaps rather a jsrelease step can be added to ploneintranet's Makefile which does all the below steps).
cd ./prototype && make jsrelease && cp LATEST ../  &&cd .. && git commit LATEST -m "Updated js release"